int32 box_id
bool use_gripper_EE
---
